source/user-manual/reference/internal-options.rst

```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | description    | lru cache size assigned for remediations' os data (in number of elements).       |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | default value  | 2048                                                                             |
++                                                    +----------------+----------------------------------------------------------------------------------+
+|                                                    | allowed values | any integer between 1 and 100000                                                 |
```

Comment:
`description`, `default`, `any`, and `allowed` should be capitalized, please check the rest of the table and follow the same style

Reply:
Done. Thanks